.github/workflows/gradle.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will build a Java project with Gradle and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-java-with-gradle
+
+name: Java CI with Gradle
+
+on: [push, pull_request]
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        token: ${{ secrets.TOKEN_GIT }}
+    - name: Set up JDK 11
+      uses: actions/setup-java@v3
+      with:
+        java-version: '11'
+        distribution: 'temurin'
+    - name: Build with Gradle
+      uses: gradle/gradle-build-action@67421db6bd0bf253fb4bd25b31ebb98943c375e1
+      with:
+        arguments: build
+    - name: Run everything
+      run: |
+        sh run-everything
```

Comment:
I am not sure if the benchmarks should be run during development (as they take very long) or only the tests (`./gradlew check`).

Reply:
I lowered the all test runtime to 1 hour. so having these tests are okay. also added index.md for github pages, example output https://ozkanpakdil.github.io/java-json-benchmark/ 
I wish to know how to create graphics for those data but it is enough for one PR.

Reply:
The graphs published on the readme are created from google spreadsheet (link provided on the readme). The text output are converted to CSV, which are manually pasted into a new copy of the spreadsheet. This is a once or twice a year op, so it's not worth automating (but feel free to automate that)...